docs/conf.py

```diff
@@ -257,14 +259,18 @@ def clean_str(s):
     "logo": {
         "text": "HydroMT Core",
     },
-    "navbar_end": ["navbar-icon-links"],  # remove dark mode switch
+    "navbar_end": ["navbar-icon-links", "version-switcher"],  # remove dark mode switch
+    "switcher": {
+        "json_url": "https://raw.githubusercontent.com/Deltares/hydromt/doc-version-switcher/docs/_static/switcher.json",
```

Comment:
I assume this is the link you will update? Then all looks good to me :)

Reply:
indeed!